src/launchpad/kafka.py

```diff
@@ -228,7 +292,7 @@ def get_kafka_config() -> KafkaConfig:
         bootstrap_servers=bootstrap_servers,
         group_id=group_id,
         topics=topics_env.split(","),
-        concurrency=int(os.getenv("KAFKA_CONCURRENCY", "4")),
+        concurrency=int(os.getenv("KAFKA_CONCURRENCY", "2")),
         max_pending_futures=int(os.getenv("KAFKA_MAX_PENDING_FUTURES", "100")),
```

Comment:
I think max_pending_futures is unused now?

Reply:
true